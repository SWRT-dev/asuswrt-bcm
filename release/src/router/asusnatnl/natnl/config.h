#ifndef __CONFIG_DLL_H__
#define __CONFIG_DLL_H__

#include <getopt.h>
#include <natnl_lib.h>

#ifdef  __cplusplus
extern "C" {
#endif

extern char callee[128],
	registrar_uri[128];

#define DEV_ID1 "80bd8155a2540ef1e87ea2f811390e5d"
#define DEV_ID2 "ab8806d6e27ea16cd8c4557f8cb3179c"
#define SIP_SERVER "ec2-50-17-15-111.compute-1.amazonaws.com"
#define REGISTRAR_URI "sip:"SIP_SERVER
#define STUN_SERVER "stun.xten.com"
#define TURN_SERVER "numb.viagenie.ca"
#define TURN_USR "dean_li@asus.com"
#define TURN_PWD "asus"

extern int my_read_config_file(const char *filename, 
			    int *app_argc, char ***app_argv);
extern int my_parse_args(int argc, char *argv[],
					struct natnl_config *natnl_cfg,
					int *srv_port_count,
					struct natnl_srv_port srv_port_cfg[]);

#ifdef  __cplusplus
}
#endif

#endif 	/* __CONFIG_DLL_H__ */