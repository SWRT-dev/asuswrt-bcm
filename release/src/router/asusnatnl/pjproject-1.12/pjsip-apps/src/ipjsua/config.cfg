--clock-rate=8000
