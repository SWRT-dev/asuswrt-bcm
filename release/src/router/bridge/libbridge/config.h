/* libbridge/config.h.  Generated by configure.  */
/* config.h.in.  Generated automatically from configure.in by autoheader.  */

/* Define if you have libsysfs */
/* #undef HAVE_LIBSYSFS */

/* Name of package */
#define PACKAGE "bridge-utils"

/* Version number of package */
#define VERSION "1.0.6"
