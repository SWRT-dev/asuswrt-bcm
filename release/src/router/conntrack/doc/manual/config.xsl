<?xml version='1.0'?>
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
                xmlns:fo="http://www.w3.org/1999/XSL/Format"
                version="1.0">
	<xsl:param name="use.id.as.filename" select="'1'"/>
	<xsl:param name="admon.graphics" select="'1'"/>
	<xsl:param name="admon.graphics.path"></xsl:param>
	<xsl:param name="chunk.section.depth" select="0"></xsl:param>
	<xsl:param name="html.stylesheet" select="'docbook.css'"/>
</xsl:stylesheet>
