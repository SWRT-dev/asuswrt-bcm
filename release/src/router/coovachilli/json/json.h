/*
 * $Id: json.h,v 1.4 2004/08/07 03:13:52 mclark Exp $
 *
 * Copyright Metaparadigm Pte. Ltd. 2004.
 * Michael Clark <michael@metaparadigm.com>
 *
 * This library is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public (LGPL)
 * License as published by the Free Software Foundation; either
 * version 2.1 of the License, or (at your option) any later version.
 *
 * This library is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 * Lesser General Public License for more details: http://www.gnu.org/
 *
 */

#ifndef _json_h_
#define _json_h_

#include "bits.h"
#include "debug.h"
#include "linkhash.h"
#include "arraylist.h"
#include "json_util.h"
#include "json_object.h"
#include "json_tokener.h"

#endif
