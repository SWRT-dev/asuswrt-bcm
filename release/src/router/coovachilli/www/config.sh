#!/bin/sh
if [ -f config-local.sh ]; then
    . ./config-local.sh 
else
    [ -f /etc/chilli/defaults ] && . /etc/chilli/defaults
    [ -f /etc/chilli/config ]   && . /etc/chilli/config
fi
