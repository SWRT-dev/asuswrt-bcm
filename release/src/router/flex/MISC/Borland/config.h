/* $Header: /ramdisk/repositories/20_cvs_clean_up/2011-02-11_sj/src/router/flex/MISC/Borland/config.h,v 1.1.1.1 2001-04-08 23:53:37 mhuang Exp $ */
/* ------------------------------------------------ */
/* version of config.h for Borland C/C++ v4.02      */
/*                     flex version 2.5.0.6  (beta) */
/* ------------------------------------------------ */

/* Define to empty if the keyword does not work.  */
#undef const

/* Define to `unsigned' if <sys/types.h> doesn't define.  */
#undef size_t

/* Define if you have the ANSI C header files.  */
#define STDC_HEADERS 1

/* Define if you have the <malloc.h> header file.  */
#define HAVE_MALLOC_H 1

/* Define if you have the <string.h> header file.  */
#define HAVE_STRING_H 1

/* Define if you have the <sys/types.h> header file.  */
#define HAVE_SYS_TYPES_H 1

/* Define if you have <alloca.h> and it should be used (not on Ultrix).  */
#undef HAVE_ALLOCA_H

/* Define if platform-specific command line handling is necessary.  */
#undef NEED_ARGV_FIXUP
