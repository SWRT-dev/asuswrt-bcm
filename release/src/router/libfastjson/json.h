/*
 * Copyright (c) 2004, 2005 Metaparadigm Pte. Ltd.
 * Michael Clark <michael@metaparadigm.com>
 * Copyright (c) 2009 Hewlett-Packard Development Company, L.P.
 * Copyright (c) 2016 Adiscon GmbH
 * Rainer Gerhards <rgerhards@adiscon.com>
 *
 * This library is free software; you can redistribute it and/or modify
 * it under the terms of the MIT license. See COPYING for details.
 *
 */

#ifndef _fj_json_h_
#define _fj_json_h_

#ifdef __cplusplus
extern "C" {
#endif

#include "json_util.h"
#include "json_object.h"
#include "json_tokener.h"
#include "json_object_iterator.h"

/**
 * Set initial size allocation for memory when creating strings,
 * as is done for example in fjson_object_to_json_string(). The
 * default size is 32, which is very conservative. If an app
 * knows it typically deals with larger strings, performance
 * can be improved by setting the initial size to a different
 * number, e.g. 1k. Note that this also means that memory
 * consumption can increase. How far entriely depens on the
 * application and its use of json-c.
 *
 * Note: each time this function is called, the initial size is
 * changed to the given value. Already existing elements are not
 * affected. This function is usually meant to be called just once
 * at start of an application, but there is no harm calling it more
 * than once. Note that the function is NOT thread-safe and must not
 * be called on different threads concurrently.
 *
 * @param size new initial size for printbuf (formatting buffer)
 */
extern void fjson_global_set_printbuf_initial_size(int size);

/**
 * Set case sensitive/insensitive comparison mode. If set to 0,
 * comparisons for JSON keys will be case-insensitive. Otherwise,
 * they will be case-sensitive.
 * NOTE: the JSON standard demands case sensitivity. By turning
 * this off, the JSON standard is not obeyed. Most importantly,
 * if keys exists which only differ in case, only partial data
 * access is possible. So use with care and only if you know
 * exactly what you are doing!
 */
extern void fjson_global_do_case_sensitive_comparison(const int newval);

/**
 * report the current libfastjson version
 */
extern const char *fjson_version(void);

/**
 * default string hash function
 */
#define FJSON_STR_HASH_DFLT 0

/**
 * perl-like string hash function
 */
#define FJSON_STR_HASH_PERLLIKE 1

#ifndef FJSON_NATIVE_API_ONLY
#define JSON_C_STR_HASH_PERLLIKE FJSON_STR_HASH_PERLLIKE
#define json_global_set_string_hash(x) /**<< no longer exists nor is needed */
#define fjson_global_set_string_hash(x) /**<< no longer exists nor is needed */
#endif

#ifdef __cplusplus
}
#endif

#endif
