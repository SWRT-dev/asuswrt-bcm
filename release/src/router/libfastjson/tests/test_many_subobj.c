/* libfastjson testbench tool
 *
 * Copyright (c) 2016 Adiscon GmbH
 * Rainer Gerhards <rgerhards@adiscon.com>
 *
 * This library is free software; you can redistribute it and/or modify
 * it under the terms of the MIT license. See COPYING for details.
 *
 */
#include "config.h"

#include "../json.h"

#include <stdio.h>
#include <stdlib.h>
#include <string.h>

#define NUM_SUBOBJ 200
#define NUM_SUBOBJ_HALF (NUM_SUBOBJ/2)
#define NUM_SUBOBJ_QUARTER (NUM_SUBOBJ/4)

int main(int __attribute__((unused)) argc, char __attribute__((unused)) **argv)
{
	int i;
	char pb[64];
	struct fjson_object *json = fjson_object_new_object();
	if (json == NULL) {
		perror("malloc ptr table failed:");
		exit(1);
	}

	/* add some keys */
	for (i = 0 ; i < NUM_SUBOBJ ; ++i) {
		snprintf(pb, sizeof(pb), "key-%d", i);
		fjson_object_object_add_ex(json, pb, fjson_object_new_int(i), 0);
		
	}
	printf("STEP1: %s\n", fjson_object_to_json_string(json));

	/* delete some keys */
	for (i = NUM_SUBOBJ_HALF - NUM_SUBOBJ_QUARTER ;
		i < NUM_SUBOBJ_HALF + NUM_SUBOBJ_QUARTER ;
		++i) {
		snprintf(pb, sizeof(pb), "key-%d", i);
		fjson_object_object_del(json, pb);
	}
	printf("STEP2: %s\n", fjson_object_to_json_string(json));

	/* add new keys */
	for (i = NUM_SUBOBJ_HALF + NUM_SUBOBJ_QUARTER - 1;
		i >= NUM_SUBOBJ_HALF - NUM_SUBOBJ_QUARTER ;
		--i) {
		snprintf(pb, sizeof(pb), "KEY-%d", i);
		fjson_object_object_add_ex(json, pb, fjson_object_new_int(i), 0);
	}
	printf("STEP3: %s\n", fjson_object_to_json_string(json));

	/* delete the new keys again, and also update key values */
	for (i = NUM_SUBOBJ_HALF - NUM_SUBOBJ_QUARTER ;
		i < NUM_SUBOBJ_HALF + NUM_SUBOBJ_QUARTER ;
		++i) {
		snprintf(pb, sizeof(pb), "KEY-%d", i);
		fjson_object_object_del(json, pb);
	}
	for (i = 0 ; i < NUM_SUBOBJ ; ++i) {
		snprintf(pb, sizeof(pb), "key-%d", i);
		fjson_object_object_add_ex(json, pb, fjson_object_new_int(i*10), 0);
		
	}
	printf("STEP4: %s\n", fjson_object_to_json_string(json));

	/* add one more key to see that adding works when extending the array */
	snprintf(pb, sizeof(pb), "key-%d", NUM_SUBOBJ);
	fjson_object_object_add(json, pb, fjson_object_new_int(NUM_SUBOBJ));
	printf("STEP5:%s\n", fjson_object_to_json_string(json));

	fjson_object_put(json);
	return 0;
}
