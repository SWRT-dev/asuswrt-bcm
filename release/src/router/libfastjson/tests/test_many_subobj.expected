STEP1: { "key-0": 0, "key-1": 1, "key-2": 2, "key-3": 3, "key-4": 4, "key-5": 5, "key-6": 6, "key-7": 7, "key-8": 8, "key-9": 9, "key-10": 10, "key-11": 11, "key-12": 12, "key-13": 13, "key-14": 14, "key-15": 15, "key-16": 16, "key-17": 17, "key-18": 18, "key-19": 19, "key-20": 20, "key-21": 21, "key-22": 22, "key-23": 23, "key-24": 24, "key-25": 25, "key-26": 26, "key-27": 27, "key-28": 28, "key-29": 29, "key-30": 30, "key-31": 31, "key-32": 32, "key-33": 33, "key-34": 34, "key-35": 35, "key-36": 36, "key-37": 37, "key-38": 38, "key-39": 39, "key-40": 40, "key-41": 41, "key-42": 42, "key-43": 43, "key-44": 44, "key-45": 45, "key-46": 46, "key-47": 47, "key-48": 48, "key-49": 49, "key-50": 50, "key-51": 51, "key-52": 52, "key-53": 53, "key-54": 54, "key-55": 55, "key-56": 56, "key-57": 57, "key-58": 58, "key-59": 59, "key-60": 60, "key-61": 61, "key-62": 62, "key-63": 63, "key-64": 64, "key-65": 65, "key-66": 66, "key-67": 67, "key-68": 68, "key-69": 69, "key-70": 70, "key-71": 71, "key-72": 72, "key-73": 73, "key-74": 74, "key-75": 75, "key-76": 76, "key-77": 77, "key-78": 78, "key-79": 79, "key-80": 80, "key-81": 81, "key-82": 82, "key-83": 83, "key-84": 84, "key-85": 85, "key-86": 86, "key-87": 87, "key-88": 88, "key-89": 89, "key-90": 90, "key-91": 91, "key-92": 92, "key-93": 93, "key-94": 94, "key-95": 95, "key-96": 96, "key-97": 97, "key-98": 98, "key-99": 99, "key-100": 100, "key-101": 101, "key-102": 102, "key-103": 103, "key-104": 104, "key-105": 105, "key-106": 106, "key-107": 107, "key-108": 108, "key-109": 109, "key-110": 110, "key-111": 111, "key-112": 112, "key-113": 113, "key-114": 114, "key-115": 115, "key-116": 116, "key-117": 117, "key-118": 118, "key-119": 119, "key-120": 120, "key-121": 121, "key-122": 122, "key-123": 123, "key-124": 124, "key-125": 125, "key-126": 126, "key-127": 127, "key-128": 128, "key-129": 129, "key-130": 130, "key-131": 131, "key-132": 132, "key-133": 133, "key-134": 134, "key-135": 135, "key-136": 136, "key-137": 137, "key-138": 138, "key-139": 139, "key-140": 140, "key-141": 141, "key-142": 142, "key-143": 143, "key-144": 144, "key-145": 145, "key-146": 146, "key-147": 147, "key-148": 148, "key-149": 149, "key-150": 150, "key-151": 151, "key-152": 152, "key-153": 153, "key-154": 154, "key-155": 155, "key-156": 156, "key-157": 157, "key-158": 158, "key-159": 159, "key-160": 160, "key-161": 161, "key-162": 162, "key-163": 163, "key-164": 164, "key-165": 165, "key-166": 166, "key-167": 167, "key-168": 168, "key-169": 169, "key-170": 170, "key-171": 171, "key-172": 172, "key-173": 173, "key-174": 174, "key-175": 175, "key-176": 176, "key-177": 177, "key-178": 178, "key-179": 179, "key-180": 180, "key-181": 181, "key-182": 182, "key-183": 183, "key-184": 184, "key-185": 185, "key-186": 186, "key-187": 187, "key-188": 188, "key-189": 189, "key-190": 190, "key-191": 191, "key-192": 192, "key-193": 193, "key-194": 194, "key-195": 195, "key-196": 196, "key-197": 197, "key-198": 198, "key-199": 199 }
STEP2: { "key-0": 0, "key-1": 1, "key-2": 2, "key-3": 3, "key-4": 4, "key-5": 5, "key-6": 6, "key-7": 7, "key-8": 8, "key-9": 9, "key-10": 10, "key-11": 11, "key-12": 12, "key-13": 13, "key-14": 14, "key-15": 15, "key-16": 16, "key-17": 17, "key-18": 18, "key-19": 19, "key-20": 20, "key-21": 21, "key-22": 22, "key-23": 23, "key-24": 24, "key-25": 25, "key-26": 26, "key-27": 27, "key-28": 28, "key-29": 29, "key-30": 30, "key-31": 31, "key-32": 32, "key-33": 33, "key-34": 34, "key-35": 35, "key-36": 36, "key-37": 37, "key-38": 38, "key-39": 39, "key-40": 40, "key-41": 41, "key-42": 42, "key-43": 43, "key-44": 44, "key-45": 45, "key-46": 46, "key-47": 47, "key-48": 48, "key-49": 49, "key-150": 150, "key-151": 151, "key-152": 152, "key-153": 153, "key-154": 154, "key-155": 155, "key-156": 156, "key-157": 157, "key-158": 158, "key-159": 159, "key-160": 160, "key-161": 161, "key-162": 162, "key-163": 163, "key-164": 164, "key-165": 165, "key-166": 166, "key-167": 167, "key-168": 168, "key-169": 169, "key-170": 170, "key-171": 171, "key-172": 172, "key-173": 173, "key-174": 174, "key-175": 175, "key-176": 176, "key-177": 177, "key-178": 178, "key-179": 179, "key-180": 180, "key-181": 181, "key-182": 182, "key-183": 183, "key-184": 184, "key-185": 185, "key-186": 186, "key-187": 187, "key-188": 188, "key-189": 189, "key-190": 190, "key-191": 191, "key-192": 192, "key-193": 193, "key-194": 194, "key-195": 195, "key-196": 196, "key-197": 197, "key-198": 198, "key-199": 199 }
STEP3: { "key-0": 0, "key-1": 1, "key-2": 2, "key-3": 3, "key-4": 4, "key-5": 5, "key-6": 6, "key-7": 7, "key-8": 8, "key-9": 9, "key-10": 10, "key-11": 11, "key-12": 12, "key-13": 13, "key-14": 14, "key-15": 15, "key-16": 16, "key-17": 17, "key-18": 18, "key-19": 19, "key-20": 20, "key-21": 21, "key-22": 22, "key-23": 23, "key-24": 24, "key-25": 25, "key-26": 26, "key-27": 27, "key-28": 28, "key-29": 29, "key-30": 30, "key-31": 31, "key-32": 32, "key-33": 33, "key-34": 34, "key-35": 35, "key-36": 36, "key-37": 37, "key-38": 38, "key-39": 39, "key-40": 40, "key-41": 41, "key-42": 42, "key-43": 43, "key-44": 44, "key-45": 45, "key-46": 46, "key-47": 47, "key-48": 48, "key-49": 49, "KEY-149": 149, "KEY-148": 148, "KEY-147": 147, "KEY-146": 146, "KEY-145": 145, "KEY-144": 144, "KEY-143": 143, "KEY-142": 142, "KEY-141": 141, "KEY-140": 140, "KEY-139": 139, "KEY-138": 138, "KEY-137": 137, "KEY-136": 136, "KEY-135": 135, "KEY-134": 134, "KEY-133": 133, "KEY-132": 132, "KEY-131": 131, "KEY-130": 130, "KEY-129": 129, "KEY-128": 128, "KEY-127": 127, "KEY-126": 126, "KEY-125": 125, "KEY-124": 124, "KEY-123": 123, "KEY-122": 122, "KEY-121": 121, "KEY-120": 120, "KEY-119": 119, "KEY-118": 118, "KEY-117": 117, "KEY-116": 116, "KEY-115": 115, "KEY-114": 114, "KEY-113": 113, "KEY-112": 112, "KEY-111": 111, "KEY-110": 110, "KEY-109": 109, "KEY-108": 108, "KEY-107": 107, "KEY-106": 106, "KEY-105": 105, "KEY-104": 104, "KEY-103": 103, "KEY-102": 102, "KEY-101": 101, "KEY-100": 100, "KEY-99": 99, "KEY-98": 98, "KEY-97": 97, "KEY-96": 96, "KEY-95": 95, "KEY-94": 94, "KEY-93": 93, "KEY-92": 92, "KEY-91": 91, "KEY-90": 90, "KEY-89": 89, "KEY-88": 88, "KEY-87": 87, "KEY-86": 86, "KEY-85": 85, "KEY-84": 84, "KEY-83": 83, "KEY-82": 82, "KEY-81": 81, "KEY-80": 80, "KEY-79": 79, "KEY-78": 78, "KEY-77": 77, "KEY-76": 76, "KEY-75": 75, "KEY-74": 74, "KEY-73": 73, "KEY-72": 72, "KEY-71": 71, "KEY-70": 70, "KEY-69": 69, "KEY-68": 68, "KEY-67": 67, "KEY-66": 66, "KEY-65": 65, "KEY-64": 64, "KEY-63": 63, "KEY-62": 62, "KEY-61": 61, "KEY-60": 60, "KEY-59": 59, "KEY-58": 58, "KEY-57": 57, "KEY-56": 56, "KEY-55": 55, "KEY-54": 54, "KEY-53": 53, "KEY-52": 52, "KEY-51": 51, "KEY-50": 50, "key-150": 150, "key-151": 151, "key-152": 152, "key-153": 153, "key-154": 154, "key-155": 155, "key-156": 156, "key-157": 157, "key-158": 158, "key-159": 159, "key-160": 160, "key-161": 161, "key-162": 162, "key-163": 163, "key-164": 164, "key-165": 165, "key-166": 166, "key-167": 167, "key-168": 168, "key-169": 169, "key-170": 170, "key-171": 171, "key-172": 172, "key-173": 173, "key-174": 174, "key-175": 175, "key-176": 176, "key-177": 177, "key-178": 178, "key-179": 179, "key-180": 180, "key-181": 181, "key-182": 182, "key-183": 183, "key-184": 184, "key-185": 185, "key-186": 186, "key-187": 187, "key-188": 188, "key-189": 189, "key-190": 190, "key-191": 191, "key-192": 192, "key-193": 193, "key-194": 194, "key-195": 195, "key-196": 196, "key-197": 197, "key-198": 198, "key-199": 199 }
STEP4: { "key-0": 0, "key-1": 10, "key-2": 20, "key-3": 30, "key-4": 40, "key-5": 50, "key-6": 60, "key-7": 70, "key-8": 80, "key-9": 90, "key-10": 100, "key-11": 110, "key-12": 120, "key-13": 130, "key-14": 140, "key-15": 150, "key-16": 160, "key-17": 170, "key-18": 180, "key-19": 190, "key-20": 200, "key-21": 210, "key-22": 220, "key-23": 230, "key-24": 240, "key-25": 250, "key-26": 260, "key-27": 270, "key-28": 280, "key-29": 290, "key-30": 300, "key-31": 310, "key-32": 320, "key-33": 330, "key-34": 340, "key-35": 350, "key-36": 360, "key-37": 370, "key-38": 380, "key-39": 390, "key-40": 400, "key-41": 410, "key-42": 420, "key-43": 430, "key-44": 440, "key-45": 450, "key-46": 460, "key-47": 470, "key-48": 480, "key-49": 490, "key-50": 500, "key-51": 510, "key-52": 520, "key-53": 530, "key-54": 540, "key-55": 550, "key-56": 560, "key-57": 570, "key-58": 580, "key-59": 590, "key-60": 600, "key-61": 610, "key-62": 620, "key-63": 630, "key-64": 640, "key-65": 650, "key-66": 660, "key-67": 670, "key-68": 680, "key-69": 690, "key-70": 700, "key-71": 710, "key-72": 720, "key-73": 730, "key-74": 740, "key-75": 750, "key-76": 760, "key-77": 770, "key-78": 780, "key-79": 790, "key-80": 800, "key-81": 810, "key-82": 820, "key-83": 830, "key-84": 840, "key-85": 850, "key-86": 860, "key-87": 870, "key-88": 880, "key-89": 890, "key-90": 900, "key-91": 910, "key-92": 920, "key-93": 930, "key-94": 940, "key-95": 950, "key-96": 960, "key-97": 970, "key-98": 980, "key-99": 990, "key-100": 1000, "key-101": 1010, "key-102": 1020, "key-103": 1030, "key-104": 1040, "key-105": 1050, "key-106": 1060, "key-107": 1070, "key-108": 1080, "key-109": 1090, "key-110": 1100, "key-111": 1110, "key-112": 1120, "key-113": 1130, "key-114": 1140, "key-115": 1150, "key-116": 1160, "key-117": 1170, "key-118": 1180, "key-119": 1190, "key-120": 1200, "key-121": 1210, "key-122": 1220, "key-123": 1230, "key-124": 1240, "key-125": 1250, "key-126": 1260, "key-127": 1270, "key-128": 1280, "key-129": 1290, "key-130": 1300, "key-131": 1310, "key-132": 1320, "key-133": 1330, "key-134": 1340, "key-135": 1350, "key-136": 1360, "key-137": 1370, "key-138": 1380, "key-139": 1390, "key-140": 1400, "key-141": 1410, "key-142": 1420, "key-143": 1430, "key-144": 1440, "key-145": 1450, "key-146": 1460, "key-147": 1470, "key-148": 1480, "key-149": 1490, "key-150": 1500, "key-151": 1510, "key-152": 1520, "key-153": 1530, "key-154": 1540, "key-155": 1550, "key-156": 1560, "key-157": 1570, "key-158": 1580, "key-159": 1590, "key-160": 1600, "key-161": 1610, "key-162": 1620, "key-163": 1630, "key-164": 1640, "key-165": 1650, "key-166": 1660, "key-167": 1670, "key-168": 1680, "key-169": 1690, "key-170": 1700, "key-171": 1710, "key-172": 1720, "key-173": 1730, "key-174": 1740, "key-175": 1750, "key-176": 1760, "key-177": 1770, "key-178": 1780, "key-179": 1790, "key-180": 1800, "key-181": 1810, "key-182": 1820, "key-183": 1830, "key-184": 1840, "key-185": 1850, "key-186": 1860, "key-187": 1870, "key-188": 1880, "key-189": 1890, "key-190": 1900, "key-191": 1910, "key-192": 1920, "key-193": 1930, "key-194": 1940, "key-195": 1950, "key-196": 1960, "key-197": 1970, "key-198": 1980, "key-199": 1990 }
STEP5:{ "key-0": 0, "key-1": 10, "key-2": 20, "key-3": 30, "key-4": 40, "key-5": 50, "key-6": 60, "key-7": 70, "key-8": 80, "key-9": 90, "key-10": 100, "key-11": 110, "key-12": 120, "key-13": 130, "key-14": 140, "key-15": 150, "key-16": 160, "key-17": 170, "key-18": 180, "key-19": 190, "key-20": 200, "key-21": 210, "key-22": 220, "key-23": 230, "key-24": 240, "key-25": 250, "key-26": 260, "key-27": 270, "key-28": 280, "key-29": 290, "key-30": 300, "key-31": 310, "key-32": 320, "key-33": 330, "key-34": 340, "key-35": 350, "key-36": 360, "key-37": 370, "key-38": 380, "key-39": 390, "key-40": 400, "key-41": 410, "key-42": 420, "key-43": 430, "key-44": 440, "key-45": 450, "key-46": 460, "key-47": 470, "key-48": 480, "key-49": 490, "key-50": 500, "key-51": 510, "key-52": 520, "key-53": 530, "key-54": 540, "key-55": 550, "key-56": 560, "key-57": 570, "key-58": 580, "key-59": 590, "key-60": 600, "key-61": 610, "key-62": 620, "key-63": 630, "key-64": 640, "key-65": 650, "key-66": 660, "key-67": 670, "key-68": 680, "key-69": 690, "key-70": 700, "key-71": 710, "key-72": 720, "key-73": 730, "key-74": 740, "key-75": 750, "key-76": 760, "key-77": 770, "key-78": 780, "key-79": 790, "key-80": 800, "key-81": 810, "key-82": 820, "key-83": 830, "key-84": 840, "key-85": 850, "key-86": 860, "key-87": 870, "key-88": 880, "key-89": 890, "key-90": 900, "key-91": 910, "key-92": 920, "key-93": 930, "key-94": 940, "key-95": 950, "key-96": 960, "key-97": 970, "key-98": 980, "key-99": 990, "key-100": 1000, "key-101": 1010, "key-102": 1020, "key-103": 1030, "key-104": 1040, "key-105": 1050, "key-106": 1060, "key-107": 1070, "key-108": 1080, "key-109": 1090, "key-110": 1100, "key-111": 1110, "key-112": 1120, "key-113": 1130, "key-114": 1140, "key-115": 1150, "key-116": 1160, "key-117": 1170, "key-118": 1180, "key-119": 1190, "key-120": 1200, "key-121": 1210, "key-122": 1220, "key-123": 1230, "key-124": 1240, "key-125": 1250, "key-126": 1260, "key-127": 1270, "key-128": 1280, "key-129": 1290, "key-130": 1300, "key-131": 1310, "key-132": 1320, "key-133": 1330, "key-134": 1340, "key-135": 1350, "key-136": 1360, "key-137": 1370, "key-138": 1380, "key-139": 1390, "key-140": 1400, "key-141": 1410, "key-142": 1420, "key-143": 1430, "key-144": 1440, "key-145": 1450, "key-146": 1460, "key-147": 1470, "key-148": 1480, "key-149": 1490, "key-150": 1500, "key-151": 1510, "key-152": 1520, "key-153": 1530, "key-154": 1540, "key-155": 1550, "key-156": 1560, "key-157": 1570, "key-158": 1580, "key-159": 1590, "key-160": 1600, "key-161": 1610, "key-162": 1620, "key-163": 1630, "key-164": 1640, "key-165": 1650, "key-166": 1660, "key-167": 1670, "key-168": 1680, "key-169": 1690, "key-170": 1700, "key-171": 1710, "key-172": 1720, "key-173": 1730, "key-174": 1740, "key-175": 1750, "key-176": 1760, "key-177": 1770, "key-178": 1780, "key-179": 1790, "key-180": 1800, "key-181": 1810, "key-182": 1820, "key-183": 1830, "key-184": 1840, "key-185": 1850, "key-186": 1860, "key-187": 1870, "key-188": 1880, "key-189": 1890, "key-190": 1900, "key-191": 1910, "key-192": 1920, "key-193": 1930, "key-194": 1940, "key-195": 1950, "key-196": 1960, "key-197": 1970, "key-198": 1980, "key-199": 1990, "key-200": 200 }
